### DISCUSSION TITLE
Exploring MeetHour!

### DISCUSSION THREAD

**Discussion by moesaid** [v-empower/MeetHour-Web-MobileSDKs#55] [General]
Could you clarify if MeetHour functions as a layer built on top of Jitsi Meet? This query stems from the observation that both the SDK and exposed API of MeetHour closely resemble those of Jitsi Meet. If MeetHour does indeed build upon Jitsi Meet, could you outline any additional features or functionalities that MeetHour offers that are not present in the base Jitsi Meet platform?

**Comment by fareed069417** [comment]
@moesaid - Meet Hour is very similar to Video Conference tools such as Zoom, Google Meet, Agora, Jitsi Meet, Twilio, etc with additional features such as Upto 3 Parallel Live Streaming, Scheduling Meeting APIs, Integration in Flutter, Flutter Web, React Native, PHP, Laravel, Run more than 1000 Parallel calls depending on licences and all will be under Auto Start Recording. We have customers who have integrated Meet Hour within their platform and have sucessfully completed more than 100K meeting with server side recording. For reference look at this - https://www.youtube.com/watch?v=eyeoVfA9pek

For more details about features of Meet Hour, check out our presentation -  https://bit.ly/meethour-ppt
